quantmsio/core/quantms/mztab.py

```
@@ -0,0 +1,808 @@
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        """Context manager exit with cleanup."""
+        self.cleanup()
+
+    def cleanup(self):
+        """Clean up resources."""
+        if self._duckdb:
+            try:
+                self._duckdb.close()
+            except:
+                pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)